contrib/containerized-pbench/README.md

```diff
@@ -1,81 +1,118 @@
-# containerized-pbench
-
-## Openshift
-
-We can either build an image using the following commands or pull the image from dockerhub
-
-Build the image from source like
+## Containerized pbench
+This directory contains OPenShift templates, dockerfiles needed to build and run containerized pbench.
 
+### Prepare the nodes you want to monitor
+The nodes need to have pbench-agent, collectd images on it. The qcow images generated by the image provisioner already has the images baked in. In case the nodes doesn't have the images, please pull them from dockerhub:
+#### pbench-agent
 ```
-$ cd containerized-pbench/dockerfiles-pod
-$ docker build -t pbench-agent .
+$ docker pull ravielluri/image:agent
+$ docker tag ravielluri/image:agent pbench-agent:latest
 ```
 
-Pull the image from dockerhub, if the image is not present on the openshift nodes, it is pulled automatically from dockerhub or registry
+### Label the nodes with a type=pbench label
+```   
+$ oc label node <node> type=pbench
+```
 
+### Create a service account and add it to the privileged scc
```

Comment:
What's an scc?

Reply:
Security Context Constraints. I will update the PR with the full form.